### ISSUE TITLE
Deploying Node.js apps from another Windows drive than C: crashes SDK

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#258]
```
What steps will reproduce the problem?
From Windows: Deploy Hello World example from a folder on another drive than C:
(gcloud preview app deploy app.yaml --set-default)

What is the expected output? What do you see instead?
Same as deploying from C:\someFolder\

What is the output of 'gcloud info'?
Google Cloud SDK [0.9.78]

Platform: [Windows, x86_64]
Python Version: [2.7.9 (default, Dec 10 2014, 12:24:55) [MSC v.1500 32 bit (Inte
l)]]
Site Packages: [Disabled]

Installation Root: [C:\Users\ThisUser\AppData\Local\Google\Cloud 
SDK\google-cloud
-sdk]
Installed Components:
  core: [2015.09.11]
  bq-win: [2.0.18]
  core-win: [2015.08.10]
  app: [2015.09.11]
  gcloud: [2015.09.11]
  windows-ssh-tools: [2015.06.02]
  gsutil: [4.15]
  bq: [2.0.18]
  preview: [2015.09.03]
  gsutil-win: [4.13]
System PATH: [C:\Users\ThisUser\AppData\Local\Google\Cloud 
SDK\google-cloud-sdk\b
in\..\bin\sdk;TRUE;C:\Users\ThisUser\AppData\Local\Google\Cloud 
SDK\google-cloud-
sdk\bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\NVIDIA Corporati
on\PhysX\Common;C:\Program Files\Common Files\Microsoft Shared\Windows Live;C:\P
rogram Files (x86)\Common Files\Microsoft Shared\Windows Live;C:\Program Files\N
VIDIA GPU Computing Toolkit\CUDA\v5.0\bin\;C:\Program Files\NVIDIA GPU Computing
 Toolkit\CUDA\v5.0\libnvvp\;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program
 Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbe
m;C:\Program Files (x86)\Intel\OpenCL SDK\2.0\bin\x86;C:\Program Files (x86)\Int
el\OpenCL SDK\2.0\bin\x64;C:\Program Files\Intel\Intel(R) Management Engine Comp
onents\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\P
rogram Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Fi
les (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel
\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;c:\Program Files
(x86)\Microsoft SQL Server\100\Tools\Binn\;c:\Program Files\Microsoft SQL Server
\100\Tools\Binn\;c:\Program Files\Microsoft SQL Server\100\DTS\Binn\;C:\Program
Files\MATLAB\R2010a\runtime\win64;C:\Program Files\MATLAB\R2010a\bin;C:\Program
Files\D3DGear;C:\Program Files (x86)\Windows Live\Shared;C:\Program Files\nodejs
\;C:\Program Files\TortoiseGit\bin;C:\Program Files (x86)\Windows Kits\8.1\Windo
ws Performance Toolkit\;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:
\Program Files (x86)\Microsoft SDKs\TypeScript\1.0\;C:\Windows\System32\WindowsP
owerShell\v1.0\;C:\Program Files (x86)\Skype\Phone\;C:\Ruby22-x64\bin;C:\Ruby200
-x64\bin;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\Wi
relessCommon\;C:\Program Files\Java\jdk1.6.0_45\bin;C:\Program Files\Java\jre7\b
in;C:\Users\ThisUser\Downloads\apache-maven-3.0.5-bin\apache-maven-3.0.5\bin;C:\
U
sers\ThisUser\Downloads\apache-ant-1.9.2-bin\apache-ant-1.9.2\bin;C:\Program 
File
s (x86)\Git\bin;C:\Program Files (x86)\CMake 2.8\bin;C:\gradle-1_11\bin;C:\Users
\ThisUser\AppData\Roaming\npm;;]
Cloud SDK on PATH: [True]

Installation Properties: [C:\Users\ThisUser\AppData\Local\Google\Cloud 
SDK\google
-cloud-sdk\properties]
User Config Directory: [C:\Users\ThisUser\AppData\Roaming\gcloud]
User Properties: [C:\Users\ThisUser\AppData\Roaming\gcloud\properties]
Current Workspace: [None]
Workspace Config Directory: [None]
Workspace Properties: [None]

Account: [current@account.com]
Project: [current-project]

Current Properties:
  [core]
    project: [current-project]
    account: [current@account.com]
    disable_usage_reporting: [True]

Logs Directory: [C:\Users\ThisUser\AppData\Roaming\gcloud\logs]
Last Log File: 
[C:\Users\ThisUser\AppData\Roaming\gcloud\logs\2015.09.17\23.07.03
.118000.log]
Please provide any additional information below.
Provisioning remote build service.
Traceback (most recent call last):
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\gcloud\gcloud.py", line 231, in main
    _cli.Execute()
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\calliope\cli.py", line 580, in Execute
    result = args.cmd_func(cli=self, args=args)
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\calliope\backend.py", line 1129, in Run
    result = command_instance.Run(args)
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\appengine\app_commands\deploy.py", line 208, in Run
    implicit_remote_build)
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\appengine\lib\deploy_command_util.py", line 137, in BuildAn
dPushDockerImages
    with docker_util.DockerHost(cli, version_id, remote) as docker_client:
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\appengine\lib\images\docker_util.py", line 208, in __enter_
_
    self._vm = Provision(self._cli, self._name, self._zone)
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\appengine\lib\images\docker_util.py", line 51, in Provision

    vm = _CreateVm(cli, name, zone)
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\appengine\lib\images\docker_util.py", line 106, in _CreateV
m
    return _Vm(cli, vm_info[0])
  File "C:\Users\ThisUser\AppData\Local\Google\Cloud SDK\google-cloud-sdk\bin\..\
./lib\googlecloudsdk\appengine\lib\images\docker_util.py", line 125, in __init__

    self.cert_dir = os.path.relpath(tempfile.mkdtemp())
  File "C:\Python27\lib\ntpath.py", line 528, in relpath
    % (path_prefix, start_prefix))
ValueError: path is on drive c:, start on drive I:

If you would like to report this issue, please run the following command:
  gcloud feedback
```

Original issue reported on code.google.com by `jahrens...@gmail.com` on 17 Sep 2015 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. We are looking at this and will get back.
```

Original comment by `gsfow...@google.com` on 23 Sep 2015 at 7:25

**Comment by GoogleCodeExporter** [comment]
```
Issue 363 has been merged into this issue.
```

Original comment by `gsfow...@google.com` on 3 Nov 2015 at 7:55

**Comment by GoogleCodeExporter** [comment]
```
Issue 366 has been merged into this issue.
```

Original comment by `gsfow...@google.com` on 3 Nov 2015 at 7:56

**Comment by GoogleCodeExporter** [comment]
```
Issue 349 has been merged into this issue.
```

Original comment by `gsfow...@google.com` on 4 Nov 2015 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
We've resolved this issue in the 0.9.87 release of the Cloud SDK (not yet 
released).

Once it's out, I'll close out this bug.
```

Original comment by `z...@google.com` on 12 Nov 2015 at 5:25
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdayal@google.com` on 19 Nov 2015 at 11:30
- Changed state: **Fixed**
